driverapi/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(
+    DRIVERAPI_SOURCES
+    ${CMAKE_SOURCE_DIR}/driverapi/src/librecuda.cpp
+    ${CMAKE_SOURCE_DIR}/driverapi/src/librecuda_status.cpp
+    ${CMAKE_SOURCE_DIR}/driverapi/src/cmdqueue.cpp
+)
+
+add_library(driverapi SHARED ${DRIVERAPI_SOURCES})
+target_include_directories(
+    driverapi PUBLIC 
+    ${CMAKE_SOURCE_DIR}/driverapi/include 
```

Comment:
include/ still contains headers that are considered internal.:
librecuda.h &
librecuda_status.h
are the ONLY two PUBLIC headers that should remain in include/.
Everything else should be moved to internal/

Reply:
OK, updated.